client/state/data-layer/wpcom/me/account/close/index.js

```diff
@@ -30,9 +30,9 @@ export function fromApi( response ) {
 	return response;
 }
 
-export function receiveAccountCloseSuccess() {
+export function receiveAccountCloseSuccess( action, response ) {
```

Comment:
You can actually address this "unused variable" ESLint warning by prefixing with `_`. It's a convention respected by our ESLint config.

```suggestion
export function receiveAccountCloseSuccess( _action, response ) {
```